app/Http/Resources/API/ListClientUserResource.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Resources\API;
+
+use App\Http\Resources\Resource;
+use Illuminate\Support\Arr;
+
+final class ListClientUserResource extends Resource
+{
+    protected function getResponse(): array
+    {
+        $resource = $this->resource;
```

Comment:
Should this class contains array of client

```suggestion
        $clients = [];

        foreach ($this->resource as $client) {
            $clients['data'][] = new ClientUserResource($client);
        }
        
        return $clients;
```